### PR TITLE
[cost] Tag Gitea workspace service in Terraform

### PR DESCRIPTION
**Pull request by ciprianmaf**
# PR for issue #1892

- As a TRE developer,I want Gitea workspace service to be tagged so that I can include costs in TRE Cost Reporting;
- Included tre_id, tre_workspace_id, tre_workspace_service_id in Gitea service.

### CODE REVIEW

**Review by tamirkamara — CHANGES_REQUESTED** [review]
There're resources in this bundle that are missing tags.
I think that at least some of them must have those (like the webapp)

**Review by tamirkamara — CHANGES_REQUESTED** [review]

File: templates/workspace_services/gitea/terraform/locals.tf [hunk 1]
```diff
@@ -11,4 +11,9 @@ locals {
   issuer                         = "https://login.microsoftonline.com/${local.aad_tenant_id}/v2.0"
   keyvault_name                  = lower("kv-${substr(local.workspace_resource_name_suffix, -20, -1)}")
   version                        = replace(replace(replace(data.local_file.version.content, "__version__ = \"", ""), "\"", ""), "\n", "")
+  workspace_service_tags = {
+    tre_id                   = var.tre_id
+    tre_workspace_id         = var.workspace_id
+    tre_workspace_service_id = var.parent_service_id
```
[the remark this answers is not in the document]
Comment (ciprianmaf) [reply templates/workspace_services/gitea/terraform/locals.tf]: modified

File: templates/workspace_services/gitea/porter.yaml [hunk 2]
```diff
@@ -18,6 +18,10 @@ parameters:
     type: string
   - name: tre_id
     type: string
+  - name: parent_service_id
+    type: string
+    description: "Resource group containing the shared ACR"
+    env: PARENT_SERVICE_ID
```
[the remark this answers is not in the document]
Comment (ciprianmaf) [reply templates/workspace_services/gitea/porter.yaml]: modified

**Review by tamirkamara — APPROVED** [review]

**Pull request merged**